config-dist.toml

```diff
@@ -30,3 +30,16 @@ urls = [
 # Default: info
 # Values: off, error, warn, info, debug, trace
 # level = "warn"
+
+[http]
```

Comment:
Should only one `[http]` block

Reply:
This config begin is a `listen` right?

Actually, this `listen` and `cors` belong `[http]`

We need changed config, add a `[http]` block, and move `listen` to `[http]`, after add `core` to http